backend/app/views/spree/admin/users/_lifetime_stats.html.erb

```diff
@@ -20,7 +20,7 @@
       </tr>
       <tr>
         <td><%= Spree.t('admin.user.store_credits') %>:</td>
-        <td id="store_credit"><%= Spree::Money.new(@user.total_available_store_credit) %></td>
+        <td id="store_credit"><%= Spree::Money.new(@user.total_available_store_credit(nil, current_store)) %></td>
```

Comment:
It's changed here: https://github.com/spree/spree/pull/11142/files#diff-cbe892c2336366a40b1f8a6f9bd28ebf8e7d1d1eb5f4a3223e475f69b841f56fR33

Reply:
@tomurb Ok, thanks. So I am closing this pr.